frontend/src/v5/ui/themes/theme.ts

```diff
@@ -982,6 +982,10 @@ export const theme = createTheme({
 						display: 'none',
 					},
 				},
+			},
+		},
+		MuiFormLabel: {
```

Comment:
Why did you need to do this?

Reply:
That is for the asterisk on the label when the input is required, so regardless whether we are using an `<InputLabel />` directly, or have it rendered via a `<Textfield />` (or other MUI inputs), the asterisk still gets the right styling